Workflow.podspec

```diff
@@ -1,6 +1,8 @@
+load('.workflow-version')
```

Comment:
using `load` can cause issues when a constant is defined and multiple podspecs are loaded into Ruby - I would use `require_relative` for this

Reply:
I was having a tough time getting `require_relative` to work with a file name `.workflow-version`. If I name it with a `.rb` extension, it works fine.

Reply:
ah yes that's possible